pages/project-versions/update/js/index.js

```diff
@@ -21,7 +21,7 @@ start({
     commentable: state.static.commentable,
     establishment: state.static.establishment,
     showComments: state.static.showComments,
-    readonly: (state.model.status !== 'draft' && state.model.status !== 'withdrawn') || !state.static.editPerms.canUpdate,
+    readonly: (state.model.status !== 'draft' && state.model.status !== 'withdrawn') || !state.static.canUpdate,
```

Comment:
Do we still need the `withdrawn`?

Is that for backwards compat?

Reply:
no, it's an oversight - good spot

Reply:
but also, I wonder if we should leave it for b/c?